server/views/pages/visitors.njk

```diff
@@ -69,17 +70,26 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
+      {% include "../partials/errorSummary.njk" %}
+
       <h1 class="govuk-heading-l">{{ pageHeaderTitle }}</h1>
 
       <p><strong class="govuk-!-margin-right-4">Prisoner name:</strong> {{ prisonerName }}</p>
+
+      {% if errors | length %}
+      <p id="visitors-error" class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span> {{ errors[0].msg }}
```

Comment:
This is hardcoded because there is only ever a single error on this page, there aren't specific ones for each field.

Reply:
Makes sense 👍